src/components/LoginForm.jsx

```diff
@@ -7,40 +7,25 @@ const apiUrl = import.meta.env.VITE_API_BACKEND_URL + "/login";
 const LoginForm = () => {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
-  const [error, setError] = useState('');
-  const [success, setSuccess] = useState('');
 
   const handleLogin = async (event) => {
     event.preventDefault();
 
-    try {
-      const response = await axios.post(apiUrl, { email, password }, {
-        headers: { 
-          "Content-Type": "application/json"
-        }
-      });
-
-      if (response.status === 200) {
-        setSuccess('Login successful!');
-        setError('');
-      } else {
-        setError('Login failed. Please try again.');
-        setSuccess('');
-      }
-    } catch (err) {
-      setError('An error occurred. Please try again.');
-      setSuccess('');
-    }
-  };
+    await axios.post(apiUrl, { email, password },  {headers: { "Content-Type": "application/json"}})
+    .then((response) => {console.log('Login successful, token: ' + response.data)})
+    .catch((error) => {console.log('There was an error in login.' , error)})
```

Comment:
Please try to avoid using `await` and `Promise.then`/`Promise.catch` at the same time. You should use one or the other.
Those are pretty much equivalent APIs, but if you mix them it will be harder to manage/read.


I would use: 
```js

axios.post(apiUrl,
    { email, password },
    { headers: { "Content-Type": "application/json" } })
    .then((response) => { console.log('Login successful, token: ' + response.data) })
    .catch((error) => { console.log('There was an error in login.', error) })
```

or the equivalent: 
```js
try {
    const response = await axios.post(apiUrl,
        { email, password },
        { headers: { "Content-Type": "application/json" } });
    console.log('Login successful, token: ' + response.data);
} catch (error) {
    console.log('There was an error in login.', error)
}
```

Reply:
Noted, fixed!

Reply:
Mark, as I removed the 'await' from the code, CI checks fail at tests with this:
'sh: 1: vitest: not found
Error: Process completed with exit code 127.'

Do you might know, why? Thank you!